test/coverlet.core.tests/Helpers/InstrumentationHelperTests.cs

```diff
@@ -34,23 +35,23 @@ public void TestGetDependenciesWithTestAssembly()
         }
 
         [Fact]
-        public void EmbeddedPortablePDPHasLocalSource_DocumentDoesNotExist_ReturnsFalse()
+        public void EmbeddedPortablePDPHasLocalSource_NoDocumentsExist_ReturnsFalse()
         {
             var fileSystem = new Mock<FileSystem> {CallBase = true};
             fileSystem.Setup(x => x.Exists(It.IsAny<string>())).Returns(false);
 
             var instrumentationHelper =
                 new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), fileSystem.Object, new Mock<ILogger>().Object, new SourceRootTranslator(typeof(InstrumentationHelperTests).Assembly.Location, new Mock<ILogger>().Object, new FileSystem()));
 
-            Assert.False(instrumentationHelper.PortablePdbHasLocalSource(typeof(InstrumentationHelperTests).Assembly.Location, out string notFoundDocument));
-            Assert.False(notFoundDocument.IsNullOrEmpty());
+            Assert.False(instrumentationHelper.PortablePdbHasLocalSource(typeof(InstrumentationHelperTests).Assembly.Location, AssemblySearchType.MissingAny));
+            Assert.False(instrumentationHelper.PortablePdbHasLocalSource(typeof(InstrumentationHelperTests).Assembly.Location, AssemblySearchType.MissingAll));
         }
 
         [Fact]
-        public void EmbeddedPortablePDPHasLocalSource_AllDocumentsExist_ReturnsTrue()
+        public void EmbeddedPortablePDPWithAssemblySearchTypeMissingAny_AllDocumentsExist_ReturnsTrue()
```

Comment:
Since both `MissingAny` and `MissingAll` are tested:

```suggestion
        public void EmbeddedPortablePDPHasLocalSource_AllDocumentsExist_ReturnsTrue()
```

It would also be good with a testcase where one file is missing, to test the difference in behaviour betweeen the two cases.

Reply:
added test with one missing document